.travis.yml

```diff
@@ -50,7 +50,7 @@ install:
   # ---------------------------------
   - conda config --add channels scitools
   - conda install --quiet --file conda-requirements.txt
-  - conda install --quiet -c scitools/channel/dev biggus=0.7.0.post.180 --no-deps --yes
+  - conda install --quiet -c scitools/channel/dev biggus=0.8.0 --no-deps --yes
```

Comment:
I'd have thought we could remove this altogether as its covered by the `conda-requirements.txt` Am I missing something?